etcdctl/ctlv3/command/global.go

```diff
@@ -138,6 +148,12 @@ func newClientCfg(endpoints []string, dialTimeout time.Duration, scfg *secureCfg
 	if scfg.insecureSkipVerify && cfg.TLS != nil {
 		cfg.TLS.InsecureSkipVerify = true
 	}
+
+	if acfg != nil {
+		cfg.User = acfg.user
```

Comment:
user and username is different. user can contain both username and optional password.